### PR TITLE
GET on inventory: update service logic

### PR DESCRIPTION
**Pull request by shivacharanms**

### CODE REVIEW

File: svc-update/update/getresources.go [hunk 1]
```diff
@@ -87,3 +95,161 @@ func GetUpdateService() response.RPC {
...
+		"Transfer-Encoding": "chunked",
+		"OData-Version":     "4.0",
+	}
+
+	requestData := strings.Split(req.ResourceID, ":")
+	if len(requestData) <= 1 {
+		errorMessage := "error: SystemUUID not found"
+		return common.GeneralError(http.StatusNotFound, response.ResourceNotFound, errorMessage, []interface{}{"SoftwareInventory", req.ResourceID}, nil)
+	}
+	data, gerr := e.DB.GetResource("SoftwareInventory", req.URL)
```
Comment (jeevan-kamkar) [inline svc-update/update/getresources.go:236]: While getting the System, Chassis or Manager Resource, if it is not found in DB the  resource, will be fetched from the device and stored again please confirm once whether that logic needs to be added here

File: svc-update/update/getresources.go [hunk 2]
```diff
@@ -87,3 +95,161 @@ func GetUpdateService() response.RPC {
...
+	requestData := strings.Split(req.ResourceID, ":")
+	if len(requestData) <= 1 {
+		errorMessage := "error: SystemUUID not found"
+		return common.GeneralError(http.StatusNotFound, response.ResourceNotFound, errorMessage, []interface{}{"FirmwareInventory", req.ResourceID}, nil)
+	}
+	data, gerr := e.DB.GetResource("FirmwareInventory", req.URL)
+	if gerr != nil {
+		log.Printf("error getting firmware inventory details : %v", gerr.Error())
+		errorMessage := gerr.Error()
+		if errors.DBKeyNotFound == gerr.ErrNo() {
```
Comment (jeevan-kamkar) [inline svc-update/update/getresources.go:161]: While getting the System, Chassis or Manager Resource, if it is not found in DB the  resource, will be fetched from the device and stored again please confirm once whether that logic needs to be added here

**Review by jeevan-kamkar — APPROVED** [review]
please cross verify and if required add the code snippet to retrieve the information from the resource if it's not found in DB  in follow up PR's
Other Changes LGTM

**Review by a-ajith — APPROVED** [review]
LGTM

**Review by bharath-b-hpe — APPROVED** [review]

File: svc-update/go.mod [hunk 3]
```diff
@@ -6,5 +6,6 @@ require (
 	github.com/ODIM-Project/ODIM/lib-dmtf v0.0.0-20200821035243-eace91cf2147
 	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20200821035243-eace91cf2147
 	github.com/ODIM-Project/ODIM/svc-plugin-rest-client v0.0.0-20200824082927-a64d9b6849af
+	github.com/ODIM-Project/ODIM/svc-systems v0.0.0-20200825050041-8ecdee28f564
```
Comment (jeevan-kamkar) [inline svc-update/go.mod:9]: remove the svc-system reference from go.mod

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Remove the svc-system reference

File: svc-update/update/getresources.go (hunk 1 above)
Comment (shivacharanms) [reply svc-update/update/getresources.go:246]: addressed

File: svc-update/update/getresources.go (hunk 2 above)
Comment (shivacharanms) [reply svc-update/update/getresources.go:162]: Addressed

File: svc-update/go.mod (hunk 3 above)
Comment (shivacharanms) [reply svc-update/go.mod]: Addressed

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

**Review by a-ajith — APPROVED** [review]

**Review by a-ajith — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Pull request merged**